com.unity.mobile.notifications/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this package will be documented in this file.
 
+## [Unreleased]
+
+### Fixes:
+- [Android] - Remove excessive things from default manifest.
```

Comment:
Would it be possible to specify what 'excessive things' are as it sounds a bit vague.

Reply:
Corrected.